### PR TITLE
[139] Product Page using HTML

### PR DESCRIPTION
**Pull request by rubyade**
HTML Product page task.
Please find screenshots below

IMAG1503
IMAG1504

### CODE REVIEW

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Weldone @rubyade Your html file is not properly named. There should be no space in the naming of files, also use small letters all through or use camel-casing. Also change RubyAdeniregun to rubyAdeniregun.

File: submissions/RubyAdeniregun/HTML/Product Page.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>DUNA STORES</title>
+</head>
+<body>
+   <hi><b>DUNA STORES</b></hi>
```
Comment (Ireti203) [inline submissions/RubyAdeniregun/HTML/Product Page.html:8]: <hi> is not an html tag. I think you wanted to use <h1> instead.

 Kindly correct this.

File: submissions/RubyAdeniregun/HTML/Product Page.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+           <li><a href="#">Back</a></li>
+           <li><a href="#">Categories</a></li>
+       </ul>
+   </nav>
+   <hr>
+   <section>
+    <h2>Fresh Body Wash (Organic and Skin Brightening Black Soap)</h2>
+    <h3><em>Brand: Skin Perfection Organics</em></h3>
+   </section>
+
```
Comment (Ireti203) [inline submissions/RubyAdeniregun/HTML/Product Page.html:21]: Remove whitespace

File: submissions/RubyAdeniregun/HTML/Product Page.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+   <!--This section is for product details-->
+   <section>
+       <h3>Product Details</h3>
+       <img src="C:\Users\CRM ABUJA\Desktop\DUFUNA\Images\IMG-20201001-WA0129.jpg" alt="Fresh Body Wash" style="width: 200px; height: 150px;">
```
Comment (Ireti203) [inline submissions/RubyAdeniregun/HTML/Product Page.html:25]: I can see that your Images folder is completely outside your project. Kindly create the Images folder within your project. You can then edit the img src appropriately.

File: submissions/RubyAdeniregun/HTML/Product Page.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+       <img src="C:\Users\CRM ABUJA\Desktop\DUFUNA\Images\IMG-20201022-WA0042.jpg" alt="Fresh Body Wash" style="width: 200px; height: 150px;">
+       <p>
+        <b> &#8358; 2,500</b>
+     </p>
```
Comment (Ireti203) [inline submissions/RubyAdeniregun/HTML/Product Page.html:29]: The closing tag is not properly formatted

File: submissions/RubyAdeniregun/HTML/Product Page.html [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+       <p>
+           Size: 200g
+        </p>
```
Comment (Ireti203) [inline submissions/RubyAdeniregun/HTML/Product Page.html:32]: same here

File: submissions/RubyAdeniregun/HTML/Product Page.html [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        <h5>Promotion:  Buy 2 and get 5% off </h5>
+
```
Comment (Ireti203) [inline submissions/RubyAdeniregun/HTML/Product Page.html:34]: Remove whitespace

File: submissions/RubyAdeniregun/HTML/Product Page.html [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+                skin. Made with natural herbs, Aloevera and Raw honey, this black soap 
+                will surely It will soothe smoothen and remove dull cells to reveal your 
+                radiant skin. It also contains lemon grass essential oil which is known 
+                to have anti-inflammatory, anti-microbial and skin brightening properties.
+                A perfectly cleansed skin, Brightened skin complexion and of course 
+                a fresh and healthy skin with consistent use is what you will get with this product.
+            </p>
+        </article>
+    </section>
+
```
Comment (Ireti203) [inline submissions/RubyAdeniregun/HTML/Product Page.html:50]: Remove whitespace

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/RubyAdeniregun/HTML/Product Page.html (hunk 1 above)
Comment (rubyade) [reply submissions/RubyAdeniregun/HTML/Product Page.html]: Yes, you are right, I meant to use the h1 tag. I have corrected it. As for the "b" tag, I wanted to use it to bolden the word "Duna Stores". Just like the strong tag. I thought the "b" tag and the strong tag do the same thing?

File: submissions/RubyAdeniregun/HTML/Product Page.html (hunk 2 above)
Comment (rubyade) [reply submissions/RubyAdeniregun/HTML/Product Page.html]: Done.

File: submissions/RubyAdeniregun/HTML/Product Page.html (hunk 3 above)
Comment (rubyade) [reply submissions/RubyAdeniregun/HTML/Product Page.html]: Done. Relocated images folder into submissions folder in the Dufuna Fem Project Folder and changed img file path

File: submissions/RubyAdeniregun/HTML/Product Page.html (hunk 7 above)
Comment (rubyade) [reply submissions/RubyAdeniregun/HTML/Product Page.html]: Done.

File: submissions/RubyAdeniregun/HTML/Product Page.html (hunk 6 above)
Comment (rubyade) [reply submissions/RubyAdeniregun/HTML/Product Page.html]: Done.

**Pull request closed without merging**